app/controllers/application_controller.rb

```diff
@@ -33,30 +35,35 @@ class ApplicationController < ActionController::Base
   end
 
   protected
-    def load_labels
-      @labels = Label.viewable_by(current_user).ordered
-    end
+  def load_labels
+    @labels = Label.viewable_by(current_user).ordered
+  end
 
-    def set_locale
-      if user_signed_in? && !current_user.locale.blank?
-        I18n.locale = current_user.locale
-      else
-        locales = []
-
-        Dir.open("#{Rails.root}/config/locales").each do |file|
-          unless ['.', '..'].include?(file)
-            # strip of .yml
-            locales << file[0...-4]
-          end
+  def set_locale
+    if user_signed_in? && !current_user.locale.blank?
+      I18n.locale = current_user.locale
+    else
+      locales = []
+
+      Dir.open("#{Rails.root}/config/locales").each do |file|
+        unless ['.', '..'].include?(file)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.